src/components/LabelMenus/styles.module.css

```diff
@@ -1,81 +1,95 @@
 .labelMenu {
-    width: 20%;
+  width: 20%;
 }
 
+.menuTop {
+  margin-top: 3.5rem;
+}
 .labelMenus {
-    height: 13rem;
-    border: solid 0.0625rem #707070;
-    background-color: #f4f4f4;
-    margin: 2.1875rem 0rem;
-    padding: 1.875rem 0rem 0rem 0rem;
+  height: 0.2rem;
```

Comment:
I don't actually see this class being used anywhere anymore, but could be missing something!

Reply:
Actually – noticing this for a few of the classes down below. Probably just a cleanup thing for later.